hack/00-osd-managed-cluster-config-production.yaml.tmpl

```
@@ -22122,7 +22122,7 @@ objects:
         - name: NodeConditionNetworkUnavailableNotification
           severity: Warning
           summary: Worker node is experiencing NetworkUnavailable
-          activeBody: NetworkUnavailable condition is true for at least one worker.
+          activeBody: NetworkUnavailable condition is true for at least one worker. Red Hat SRE will remedy if not automatically resolved.
```

```suggestion
          activeBody: NetworkUnavailable condition is true for at least one worker. Red Hat SRE will be alerted and triage issues.  If action is required, a proactive case will be opened by Red Hat.
```

I don't want to say we will resolve it, after reflecting there is a class of rare change customers can do to break networks.  We shouldn't imply it's Red Hat's responsibility in all cases.